bundles/org.openhab.binding.tacmi/src/main/java/org/openhab/binding/tacmi/internal/schema/TACmiSchemaHandler.java

```diff
@@ -185,10 +185,12 @@ private void refreshData() {
             // binding shutdown is in progress
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE);
             this.online = false;
-        } catch (final ParseException | TimeoutException | ExecutionException | RuntimeException e) {
-            // we need the stack trace here to get an idea what happened when this happens to somebody and we try to
-            // troubleshoot this
-            logger.debug("Error loading API Scheme: {} ", e.getMessage(), e);
+        } catch (final ParseException | RuntimeException e) {
+            logger.warn("Error parsing API Scheme: {} ", e.getMessage(), e);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.HANDLER_INITIALIZING_ERROR, "Error: " + e.getMessage());
+            this.online = false;
+        } catch (final TimeoutException | ExecutionException e) {
+            logger.warn("Error loading API Scheme: {} ", e.getMessage());
```

Comment:
You don't need to log this, as `updateStatus()` already does it.

Reply:
Damn, I didn't learn it. Sorry bothering you with this again... Having the logs from `updateStatus()` I changed the level to debug for the parser errors so when the stack trace is needed the user has to enable debug logging...